core/src/main/java/com/rettichlp/unicacityaddon/listener/faction/state/WantedListener.java

```diff
@@ -113,10 +115,6 @@ public void onChatReceive(ChatReceiveEvent e) {
 
             int wpAmount = getWpAmountAndDelete(targetName);
 
-            if (playerName.equals(this.unicacityAddon.player().getName())) {
```

Comment:
Das bitte drin lassen und `playerName.equals(this.unicacityAddon.player().getName())` mit `isAddonPlayer` ersetzen